tests/caesarlib.rs

```diff
@@ -55,8 +55,8 @@ mod tests {
     #[test]
     fn custom_base_sequence_correct_encipherd_and_deciphered() {
         let base = "zyxwvutsrqponmlkjihgfedcbaZYXWVUTSRQPONMLKJIHGFEDCBA";
-        let offset = 5;
         let test_word = "FooBar";
+        let offset = 5;
```

Comment:
@Natim It is a detail that's nice. Before, I tried putting it in a 'logical' order.